.zuul.yaml

```diff
@@ -14,6 +15,7 @@
     name: system-test-fedora-34
     description: Run Toolbox's system tests in Fedora 34
     timeout: 1200
+    files: &system_test_files ['data/*', 'playbooks/*', 'profile.d/*', 'src/*', 'meson.build', 'meson_options.txt', '.zuul.yaml']
```

Comment:
Please forgive my ignorance, but what does `&system_test_files` mean? Is this the syntax for defining it as a variable? In that case, wouldn't it be good to define it as part of the `system-test-fedora-rawhide` or globally, so that it doesn't get lost by mistake when Fedora 34 goes EOL?

Reply:
This is so-called YAML anchor. I was advised by Zuul developers to use it. I don't think a global "variable" can be defined in YAML. This is a kind of a middle-ground.

Reply:
Ok, I see. :)

What do you think about defining the anchor as part of the Rawhide tests?

Reply:
Sounds sensible.

Reply:
Okay! I am trying that in https://github.com/containers/toolbox/pull/971